client/tokenized-express-checkout/index.js

```diff
@@ -65,29 +65,18 @@ const fetchNewCartData = async () => {
 };
 
 const getServerSideExpressCheckoutProductData = () => {
-	const requestShipping =
-		getExpressCheckoutData( 'product' )?.needs_shipping ?? false;
 	const displayItems = (
 		getExpressCheckoutData( 'product' )?.displayItems ?? []
 	).map( ( { label, amount } ) => ( {
 		name: label,
 		amount,
 	} ) );
-	const shippingRates = requestShipping
-		? [
-				{
-					id: 'pending',
-					displayName: __( 'Pending', 'woocommerce-payments' ),
-					amount: 0,
-				},
-		  ]
-		: undefined;
```

Comment:
This "Pending" rate is now consistently added on initialization. Before, it was added only on the product pages.